sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -123,18 +118,18 @@ public TextAnalyticsServiceVersion getServiceVersion() {
      * the country hint.
      *
      * <p><strong>Code sample</strong></p>
-     * <p>Detects language in a text. Subscribes to the call asynchronously and prints out the detected language
+     * <p>Detects language in a document. Subscribes to the call asynchronously and prints out the detected language
```

Comment:
Shouldn't this still be text?

Reply:
We are all using document instead of text.